docs/how-to/required-software.md

```diff
@@ -0,0 +1,96 @@
+# How To - Required software
+
+## Introduction
+
+This guide shows you how to setup a Mac for iOS development.
+
+You will need a Mac in order to run this. It will not work on a Windows or Linux device.
+
+## Xcode
+
+You'll need an Apple ID in order to download Xcode. Download it using **[this link](https://developer.apple.com/download/applications)**. You will need to become an Apple Developer (you will be prompted to do this, and agree to their terms and conditions) - this is free.
+
+Select the latest stable version of Xcode (i.e Xcode 15.0.1, not the Beta version), then click `Download Xcode XX.X.X (2.96GB)` to download a compressed file:
```

Comment:
Given Oli's feedback, do we want to specify not to download from the App Store?

Reply:
I'm not sure how explicit we want to be with guiding people away from an App Store installation. I could say we advise using this link to have more control over the Xcode version if you'd like?